content/posts/2015-05-03_may-newsletter.md

```diff
@@ -0,0 +1,137 @@
+software for math, science and engineering. The call for papers is closes
+this week.
+
+[SciPy 2015]: http://www.scipy2015.scipy.org/
+
+
+[DjangoCon US][] (djangocon.us) <br />
+DjangoCon will be in Austin this year from September 7-9. Texas is a popular
+place for conferences this year. Financial aid is available and the call for
+proposals is open until May 15. This year DjangoCon is run by a new nonprofit,
+the Django Events Foundation North America (DEFNA) which means lower prices and
+making the event more accessible to those who need it.
+
+[DjangoCon US]: https://2015.djangocon.us/
+
+
+Jobs
+----
+
+[Software developer][] (sportechbi.wordpress.com) <br />
+Are you into data analytics and sports? Or just looking for an early stage
+startup to join? SporTech B.I. is looking to add it's first member of the
+team outside the co-founders.  As Lead Software Engineer, you'll work
+alongside the C.T.O. to build amazing business intelligence apps for sports
+organizations, starting with soccer clubs.
+
+[Software developer]: https://sportechbi.wordpress.com/job-opportunities/
+
+
+[Full Stack Developer][full-stack-dev] (brightscope.com) <br />
+Brightscope, a local financial analytics firm and big SD Python supporter is
+looking to hire a full stack Django developer to join their ever growing team.
+They are based in the UTC area.
+
+[full-stack-dev]: http://www.brightscope.com/about/careers/#job_Software_Engineer
+
+
+[Software engineer at Intellisis][software-engineer] (intellisis.com) <br />
+Intellisis is looking for you, the agile-loving, curious, technology-obsessed
+Python connoisseur ready to take on new challenges. They're a small company
+near Torrey Pines and are all-in with Python.
+
+[software-engineer]: http://www.intellisis.com/#career-wrapper
+
+
+Notes
+-----
+
+Would you like to see your job opening on our newsletter that reaches over
+1000 local Python developers? First, read our
+[job posting guidelines][job-guidelines] and then [send it to us][send-it]
+if you would like it to appear in the newsletter or you can post it
+yourself to our mailing list.
+
+The Python Software Foundation [accepts donations][accepts-donations] on our
+behalf but there is no obligation to donate in order to highlight your
+posting here. Five dollars helps and $50 helps a lot. These donations entirely
+go toward our periodic workshops as well as our meetup.com dues.
+Also, we are on [gittip][] if you're into that sort of thing.
+
+[send-it]: mailto:sandiegopython-organizers@googlegroups.com
+[job-guidelines]: http://pythonsd.org/pages/job-posting-guidelines.html
+[accepts-donations]: https://psfmember.org/civicrm/contribute/transact?reset=1&id=9
+[gittip]: https://www.gittip.com/sandiegopython/
+
+
+Did you know that San Diego Python has a chat room? There are some nice
+folks happy to help or just chat in [#sandiegopython][irc] on Freenode.
+
+[irc]: http://pythonsd.org/pages/chat-room.html
+
+We now have a [code of conduct][]!
```

Comment:
I think you're trying to express that it's important to us, and something we're proud to have. We could just say that directly. :smiley: